src/AppBar/AppBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { PropsWithChildren } from 'react'
+import styled from 'styled-components'
+
+// TODO create css type, react might have it
+interface AppBarProps {
+  fixed?: boolean
+  search?: boolean
+  shadow?: boolean
+  width?: string
+  padding?: string
+  direction?: string
+  position?: string
+  background?: string
+  theme?: object
+}
+
+export interface AppBar extends React.FC<PropsWithChildren<AppBarProps>> {
+  Item?: React.FC<PropsWithChildren<AppBarProps>>
+}
+
+const Theme = {
+  primary: '#FFA',
+  accent: '#000',
+}
+
+// TODO props per dom element as well as all for appbar
+const Wrapper = styled('div')<AppBarProps>`
```

Comment:
wrapper should be a `nav` not a `div` as this ia a landmark area. Similarly we should enforce the navigation entries be housed in a `ul` with `li` elements

Reply:
:+1: originally wanted to do ul/li and make some more general, but then realised that app bar is quite specific already, but you are right, it should be ul/li,, will update

Reply:
done